extension/main.js

```diff
@@ -13,6 +13,15 @@
 		chrome.browserAction.setTitle({
 			title: title
 		});
+
+		if (badge && window.GitHubNotify.settings.get('showDesktopNotif')) {
+			chrome.notifications.create('github-notifier', {
+				title: 'GitHub',
+				iconUrl: 'icon-128.png',
+				type: 'basic',
+				message: 'You have new unread notifications'
```

Comment:
@sindresorhus This is quite useful when you have Chrome minimized or as background window, e.g. while working on code in IDE. `Mark as read` button doesn't makes much sense, since in most cases you need to take a look on notifications before marking them as read.